docs/excel/custom-functions-custom-enums.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Custom enums for custom functions
+description: Create custom enums to use with your custom functions.
+ms.date: 06/24/2025
+ms.localizationpriority: medium
+---
+
+# Create custom enums for your custom functions
+
+:::image type="content" source="../images/custom-functions-custom-enum-autocomplete.png" alt-text="The members of a custom enum displaying in the Excel AutoComplete menu.":::
+
+With custom enums, the members in your enum display in the Excel AutoComplete menu for your custom function. The planets in the preceding screenshot are members of a custom enum. This article describes how to create custom enums and use them as parameters in your custom functions.
```

Comment:
The member-centric framing sounded really techincal. I think people will get that an enum is inherently comprised of its members. 
```suggestion
Custom enums give users Excel AutoComplete options for your custom functions. The members in your enum show up in the formula bar as suggestions. The planets in the preceding screenshot are an example of a custom enum that provides a set list. This article describes how to create custom enums and use them as parameters in your custom functions.
```